.github/stale.yml

```diff
@@ -13,9 +13,8 @@ exemptLabels:
 staleLabel: lifecycle/stale
 
 # Comment to post when marking an issue as stale. Set to `false` to disable
-markComment: >
```

Comment:
what's the difference? it looks the same to me.
![image](https://user-images.githubusercontent.com/24452340/222358139-c222b849-4a70-434a-818c-573ed736874a.png)

Reply:
Look like:
Value of
```yaml
key: >
  a
  b
```
is
```
ab
```
***
Value of
```yaml
key: |
  a
  b
```
is
```
a
b
```

Reply:
makes sense to me then.